Lesson33/gozde/lesson33-app/src/App.js

```diff
@@ -0,0 +1,22 @@
+import {Exercise1} from './Exercise1';
+import {Exercise2} from './Exercise2';
+import { Exercise3 } from './Exercise3';
+import { Exercise4 } from './Exercise4';
+import { Exercise5 } from './Exercise5';
+import { Exercise6 } from './Exercise6';
+import { Exercise7 } from './Exercise7';
```

Comment:
Exercises 8, 9 10 are not visible on the screen since they are not added here. First, import the components:
```
import { ToDoList } from './Exercise8';
import { ColorChanger } from './Exercise9';
import { Exercise10 } from './Exercise10';
```